accelerator/config.h

```diff
@@ -188,6 +188,13 @@ status_t ta_core_set(ta_core_t* const core);
  */
 void ta_core_destroy(ta_core_t* const core);
 
+/**
+ * @brief Switch logger verbose level between quiet and verbose
+ * @param quiet[in] Switch to quiet mode or not
+ * @param init[in] Initialization or not
+ */
+void ta_logger_switch(bool quiet, bool init);
```

Comment:
I wonder if this function is necessary or not. @HowJMay, how do you think?

Reply:
I personally prefer to keep this function. This function can be applied to the main function of MQTT, too.

Reply:
Relates to #589